openapi.yaml

```diff
@@ -16,176 +16,405 @@ host: api.oregonstate.edu
+              type: string
+              format: url
+              description: Location of the newly created game
+        '400':
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+    get:
+      summary: Info of a list of game
+      tags:
+        - games
+      operationId: getGames
+      parameters: 
+        - $ref: '#/parameters/authorization'
         - in: query
-          name: species
-          description: Search by pet species
-          required: false
+          name: round
           type: string
+          description: The current round of the game
+        - in: query
+          name: playerId
+          type: string
+          description: The playerId of members that are playing this game
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/gameResults'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+  /games/{gameId}:
+    get:
+      summary: Info for a specific game
+      tags:
+        - games
+      description: Get a game by its gameid
+      operationId: getGameByGameId
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/gameId'
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/gameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+    patch:
+      tags:
+        - games
+      summary: Update a game specified by its id
+      operationId: patchGameByGameId
+      consumes:
+        - application/json
+      parameters: 
+        - $ref: '#/parameters/gameId'
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the game specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/gamePatchBody'
+      responses:
+        '200':
+          description: Successfully update a game
+          schema:
+            $ref: '#/definitions/gameResult'
         '400':
-          description: Bad request
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/errors'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+    delete:
+      tags:
+      - games
+      summary: "Remove a game specified by its id"
```

Comment:
You should add something either here or in a description about how deleting the game will delete its players as well.